src/main/java/christmas/model/OrderedMenus.java

```diff
@@ -0,0 +1,72 @@
+package christmas.model;
+
+import christmas.model.exception.CategoryException;
+import christmas.model.exception.OrderException;
+import christmas.model.exception.OverQuantityException;
+import java.util.List;
+
+public class OrderedMenus {
+    private static final int MAXIMUM_ORDER_QUANTITY = 20;
+    private final List<OrderedMenu> orderedMenus;
+
+    public OrderedMenus(List<OrderedMenu> orderedMenus) {
+        validate(orderedMenus);
+        this.orderedMenus = orderedMenus;
+    }
+
+    private void validate(List<OrderedMenu> menus) {
+        validateDuplicateMane(menus);
+        validateOverQuantity(menus);
+        validateAllCategoryIsDrink(menus);
+    }
+
+    public long totalAmount() {
+        return orderedMenus.stream()
+                .mapToLong(OrderedMenu::calculateMenuPrice)
+                .sum();
+    }
+
+    public long countDessertCategory() {
+        return orderedMenus.stream()
+                .filter(OrderedMenu::isDessertCategory)
+                .mapToInt(OrderedMenu::getQuantity)
+                .sum();
+    }
+
+    public long countMainCategory() {
+        return orderedMenus.stream()
+                .filter(OrderedMenu::isMainCategory)
+                .mapToInt(OrderedMenu::getQuantity)
+                .sum();
+    }
+
+    private void validateDuplicateMane(List<OrderedMenu> menus) {
+        long uniqueMenuSize = menus.stream().distinct().count();
```

Comment:
(추천 드려요!)
stream을 사용하는 것 보다
```suggestion
             int uniqueMenuSize = new HashSet<>(menus).size();
```
HashSet을 쓰는게 더 좋은 것 같습니다 :)

Reply:
몰랐던 좋은 방식 알아갑니다..!